### ISSUE TITLE
Provider has indicated "requires replacement" for a non-existent attribute path

### ISSUE THREAD

**Issue by pst** [hashicorp/terraform-provider-google#7284]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request.
* Please do not leave _+1_ or _me too_ comments, they generate extra noise for issue followers and do not help prioritize the request.
* If you are interested in working on this issue or have submitted a pull request, please leave a comment.
* If an issue is assigned to the `modular-magician` user, it is either in the process of being autogenerated, or is planned to be autogenerated soon. If an issue is assigned to a user, that user is claiming responsibility for the issue. If an issue is assigned to `hashibot`, a community member has claimed the issue already.

### Terraform Version

Terraform: 0.12.29
Google provider: 3.39.0

### Affected Resource(s)

* google_container_cluster

### Terraform Configuration Files

The code is open source, I'm linking to the actual pull-request that causes this issue.

https://github.com/kbst/terraform-kubestack/pull/132

### Debug Output

https://gist.github.com/pst/42e31cd5b74c8a6417a42e1f5ac8ec7e

### Panic Output

Terraform fails during plan. It does not seem to create a `crash.log`.

### Expected Behavior

No error.

### Actual Behavior

Error.

### Steps to Reproduce

1. Create a cluster with the previous TF configuration that defaults to a cluster without private nodes
1. Apply the new Terraform config to that cluster with `enable_private_nodes = false` set to retain previous behavior

### References

* #0000

**Comment by pst** [comment]
@edward2a I linked the PR the entire code base is open-source. You should be able to see everything.

**Comment by edwardmedia** [comment]
@pst I see below PR which is in the review process. I am not super clear why you create this issue. If both try to address the same issue, should we use the PR for issue tracking? Let me close this issue. Please reopen it if there is a different issue we need to address. Thanks

https://github.com/kbst/terraform-kubestack/pull/132

**Comment by pst** [comment]
@edwardmedia sorry I pinged the wrong user the first time.

Not sure why you closed this. The Terraform output suggests this is a bug in the provider. The bug surfaces as part of the linked PR, yes. But the PR is not about fixing the bug.

I linked the PR, because it shows the Terraform configuration that causes this presumed provider bug. I opened this issue, because I think the bug needs to be fixed in the provider.
